shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommands.java

```diff
@@ -1066,11 +1066,13 @@ public void writeFields(DataOutput out) throws IOException {
             long lastModified = in.readLong();
             long segmentLength = in.readLong();
             long startOffset = 0;
+            long rolloverSizeBytes = 0;
```

Comment:
What exactly did you modify in this command?

FYI I don't think we need to return the rollover size in GetInfo.